node_package/src/ReactOnRailsRSC.ts

```diff
@@ -0,0 +1,89 @@
+import type { ReactElement } from 'react';
+// @ts-expect-error will define this module types later
```

Comment:
I used `renderToPipeableStream` instead of `renderToReadableStream` because it makes it easier to share error handling code with the server rendering streaming code. Lets keep the code there as all PRs will be merged together.